docs/tutorials/acme/quick-start/index.rst

```diff
@@ -351,9 +351,13 @@ install cert-manager. This example installed cert-manager into the
 .. code-block:: shell
 
     # Install the cert-manager CRDs. We must do this before installing the Helm
-    # chart in the next step
+    # chart in the next step for `release-0.7` of cert-manager:
     $ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.7/deploy/manifests/00-crds.yaml
 
+    # NOTE: If you are using an earlier release, e.g. `release-0.6`, make sure
+    # that you are using the correct release branch here and below:
+    $ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.6/deploy/manifests/00-crds.yaml
```

Comment:
`release-0.6` here should probably be `release-0.7` instead?

Reply:
Its a warning to people to use an earlier release branch if they're using an older chart! release-0.7 is reference everywhere else!

Reply:
Hm, I would prefer to not link to older versions of the manifests here in favour of making it clear to users that they can switch versions of the docs between release branches (as we already version our documentation)